test/unit/vendor/lime/lime_harnessTest.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+require_once __DIR__.'/../../../bootstrap/unit.php';
+require_once __DIR__.'/tools/TestCase.php';
+require_once __DIR__.'/tools/lime_no_colorizer.php';
```

Comment:
What is the filename case policy in this directory?

Reply:
Same as class name.